### DISCUSSION TITLE
Building a Web API

### DISCUSSION THREAD

**Discussion by fvildoso** [ValeLang/Vale#636] [General]
Hi,

I'm interested in creating a web API using the Vale programming language. Is there a library that could help me with this?

Regards.

**Comment by Verdagon** [comment]
Hey, good question. I don't know the best answer yet, but I know it'll be a lot easier when we have full Vale/Rust FFI support to more easily enable Vale web servers. It might be several months though, since there's a lot of polishing and fixing before that's finalized. I'll post back here when it's ready!

**Reply by fvildoso** [reply]
Thanks! I'll be waiting for further information 🐻
